### DISCUSSION TITLE
parameters from the ex vivo cotyledon perfusion model

### DISCUSSION THREAD

**Discussion by wanxiang227** [Open-Systems-Pharmacology/Forum#1871] [Questions & Problems]
Hi,
I have completed the ex vivo cotyledon perfusion model of the drug, and I do not know how to obtain the two parameters Dpl and Kf,m from the results. Whether it's a model output or a calculation.
Thank you.
Best regards,
wanxiang

**Comment by AndreDlm** [comment]
Hello @wanxiang227,

following the example published by [Mian et al.](https://www.frontiersin.org/journals/pediatrics/articles/10.3389/fped.2021.733520/full), Dpl and Kf,m are unknown model input parameters. Because they are unknown, they are fitted to observed data in the ex vivo cotyledon assay using the [ex vivo model in MoBi](https://github.com/Open-Systems-Pharmacology/Pregnancy-Models/tree/master/CotyledonPerfusionModel). 
After fitting the values of these parameters were used as input for a whole-body PBPK model. Specifically, the following parameters in the PBPK model were changed to the fitted values:
`Neighborhoods|PlacentaFetal_int_PlacentaFetal_cell|Paracetamol|Partition coefficient (intracellular/plasma)` --> 4.313041
`Neighborhoods|PlacentaMaternal_pls_PlacentaFetal_cell|Paracetamol|Partition coefficient (intracellular/plasma)` --> 4.313041
`Organism|Permeability factor cell-F_pls-M` --> 0.050733
`Organism|Permeability factor pls-M_cell-F` --> 0.059578

These parameters of the PBPK model might not be visible in PK-Sim, so you need to change them in MoBi.

If you do not amend the differential equation of placental transfer in the PBPK model as suggested by Mian et al., the latter two parameters (Permeability factors) won't exist in the PBPK model. In that case, the default organ permeability (applied to all organs, under path: `MOLECULE|Permeability`) will also be used for the placenta. 
To use a different (user-defined) permeability, you need to change the ordinary differential equation for placental transfer in the passive transports building block:
![image](https://github.com/user-attachments/assets/67540a69-0a26-4884-acb0-50b61db17f12)

To do so, you first need to add placental permeability as a new parameter to you model, e.g., in the compound building block, and then use this placental permeability in the differential equation shown above, e.g., something like:
`dN/dt = P_placenta*SA*C_pls*fu - P_placenta* SA* C_cell/K_cell_pls`
with P_placenta referring to the newly added parameter (make sure to adjust the path accordingly!).

As you can see in the screenshot above, `SA` refers to the surface area between `PlacentaMaternal_pls` and `PlacentaFetal_cell` which is the apical membrane of the trophoblasts and should be around 11800 cm² at term. If you only fit the permeability in your ex vivo model, you do not need to upscale it to the whole placenta, because - as shown in the equation above - it will be multiplied by `SA`. 
However, if you fit the product of P*SA in your ex vivo MoBi model, you would need to upscale that to the whole placenta (e.g., as suggested by [de Sousa Mendes et al.](https://pubmed.ncbi.nlm.nih.gov/27766562/)) and adjust the equation and parameterization in your PBPK model accordingly. 

I hope this clarifies things a bit.

Best,
André

**Reply by wanxiang227** [reply]
@AndreDlm,
Thank you for your patient reply. I have a general understanding of the process, but I am wondering how P_placenta relates to the following two parameters. 
![微信截图_20241224171417](https://github.com/user-attachments/assets/6e3b3921-2905-4553-a783-5939fdc7445d)
Merry Christmas to you in advance.
Best,
Wanxiang

**Reply by AndreDlm** [reply]
Dear @wanxiang227, 

in the model equations used by [Mian et al.](https://www.frontiersin.org/journals/pediatrics/articles/10.3389/fped.2021.733520/full), `P_placenta` is modified by these two permeability factors. `Permeability factor cell-F_pls-M` is multiplied with `P_placenta` in the part of the equation describing efflux diffusion (from fetal cell to maternal plasma) and `Permeability factor pls-M_cell-F` is multiplied with `P_placenta` in the part of the equation describing influx diffusion (from maternal plasma to fetal cell):

dN<sub>fetal_cell</sub>/dt = PermeabilityFactor<sub>pls-M_cell-F</sub> * P<sub>placenta</sub> * SA * C<sub>pls</sub> * f<sub>u</sub>  -  PermeabilityFactor<sub>cell-F_pls-M</sub> * P<sub>placenta</sub> * SA * C<sub>cell</sub>/K<sub>cell:pls</sub>

Let me know if that makes it clear!

**Reply by Xiang-227** [reply]
@AndreDlm 
Thank you for your patient reply.
I want to ask how to substitute the obtained Dp,l and Kf,m into the model.
Best,
x

**Reply by HaiXxxxxx** [reply]
Dear @AndreDlm,
I would like to ask you how to use Monolix to calculate Dcot and Kppl. Is it calculated by substituting the formula into monolix?
Thank you.
